spec/datetimeformat.html

```diff
@@ -1159,7 +1159,7 @@ <h1>Properties of Intl.DateTimeFormat Instances</h1>
       <li>[[Locale]] is a String value with the language tag of the locale whose localization is used for formatting.</li>
       <li>[[Calendar]] is a String value representing the <a href="https://unicode.org/reports/tr35/#UnicodeCalendarIdentifier">Unicode Calendar Identifier</a> used for formatting.</li>
       <li>[[NumberingSystem]] is a String value representing the <a href="https://unicode.org/reports/tr35/#UnicodeNumberSystemIdentifier">Unicode Number System Identifier</a> used for formatting.</li>
-      <li>[[TimeZone]] is a String value used for formatting that is either a time zone identifier from the IANA Time Zone Database or a UTC offset in ISO 8601 extended format.</li>
+      <li>[[TimeZone]] is a String value used for formatting that is an available time zone identifier.</li>
```

Comment:
I'm skeptical about using the term "available time zone identifier". I'd prefer to avoid it here so we can more easily remove it later from other specs if that's what we decide. 

```suggestion
      <li>[[TimeZone]] is a String value used for formatting that is either an available named time zone identifier or an offset time zone identifier.</li>
```